### DISCUSSION TITLE
Meet Your GitHub Community Team!

### DISCUSSION THREAD

**Discussion by ossanna16** [community/community#28776] [New to GitHub]
There's a group of people whose main focus is _you_, the community. We're one mighty team with a collective goal of helping you get to your solutions faster, connect with other GitHub users, and amplify your million-dollar ideas to the world. Our team is thrilled to be in this space with you, so please feel free to say hi when you see us around!

- @ossanna16, **Senior Program Manager** builds programs for new user onboarding and retention. Being one of the leaders of PyLadies London, she loves Open Source, especially the Python community. In her free time she enjoys speaking at tech conferences and mentoring women in tech as she is a strong believer in giving back and paying it forward. She's equally passionate about diversity, community outreach, and mental health. @ossanna16 still reminisces about her first year in London where she got to see the Spice Girls at Wembley Stadium :microphone: 

- You might not see @jricciardi, around often, but don’t worry — as **Senior Community Operations Manager**, he’s lurking in the background trying find new ways to make the community experience better, more seamless, and help the rest of the team make data-informed decisions. He’s a big nerd for data visualization, search engine optimization, UX design, code for good, taxonomy, and just plain ol’ saving people time. His perfect vacation involves mountains, a cabin, a body of water to paddle on, and no notifications. His favorite emoji is this little shrugging squid <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:85.95769682726204% 14.042303172737954%;background-size:5418.75% 5418.75%" data-codepoints="1f991"></span></span>.

- @tuves, **Senior Program Manager**, focuses on GitHub content for the community. She is fascinated by human connections and how easily we can build relationships with one another, especially in this digital age. She often wonders about the Venn diagram that illustrates what people know vs what people share; and how we can increase that overlap. If she could have dinner with anyone in the world? [David Sedaris](https://www.nytimes.com/2018/05/23/books/david-sedaris-calypso.html?). Her special but useless talent? Imitating the voice of Janice from Friends. 

- @ettaboyle, **Director, Community Strategy**, has been our valiant leader since March 2021. She loves this team and how we've pushed her to learn and grow in ways she hadn't ever imagined. She's committed to growing the global GitHub community and continually inspired by stories of how improving skills as a developers truly changed the trajectory of someone's life. Want to feel inspired like @ettaboyle? Check out Erica Brescia's [The Next 50 Million Developers](https://www.youtube.com/watch?v=poENiMjjG2U) keynote from ATO <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:99.94124559341951% 2.056404230317274%;background-size:5418.75% 5418.75%" data-codepoints="2728"></span></span> So where in the world would she transport herself if given the chance? The Alps, the Italian coast, Tanzania, or Japan. <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:97.94359576968273% 59.9882491186839%;background-size:5418.75% 5418.75%" data-codepoints="2708-fe0f"></span></span>

- @queenofcorgis, **Senior Community Manager**, new to learning GitHub? She'll be your guide! Lili is a regular in the New to GitHub category making sure members are set up for success as they get acquainted with all things GitHub! She considers herself a jack of all community trades with special places in her heart for engagement, member feedback, and accessibility/inclusion initiatives. When she isn't in the GitHub community, you can find Lili outdoors, she loves parks (especially the U.S. National Parks), the beach, and a good picnic :basket: . She loves to travel to anywhere and everywhere, but if she had to choose, her perfect vacation involves hiking, beautiful surroundings, delicious meals, relaxing, and no humidity. 

- Currently a little sprout dreaming of growing into a great Community Manager tree, @ghostinhershell is our **Community Manager** focusing on the Ecosystems area of the GitHub Community. @ghostinhershell has always been technically inclined, and considers her first "real" job to be when she finally made it as a Geek Squad Agent at Best Buy <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:59.9882491186839% 85.95769682726204%;background-size:5418.75% 5418.75%" data-codepoints="1f60e"></span></span>. In the SaaS space she grew her technical knowledge while also developing strong relationships with the customers that she supported along the way. When it came time to figure out where she wanted to go next in her career, she knew that something that allowed her to use her technical knowledge while also building with people would ultimately be her North Star. @ghostinhershell spends a good chunk of time with her family, volunteering in her community, pushing her old Kia to the limit (why buy a new car when this one's running great? <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:59.9882491186839% 67.97884841363103%;background-size:5418.75% 5418.75%" data-codepoints="1f605"></span></span>) and wondering why she's still buying books when she has so many that haven't been finished yet. She's also trying her hand at sewing, since she's a #shortie and everything never fits right. Finally, she's a big fan of the self-help and self-empowerment space on YouTube. Here's a vid recommendation, free of charge: [7 life changing mindset habits to soothe the anxious soul & live lighter](https://www.youtube.com/watch?v=FmBRB2Wq8VU).

- @samus-aran, Education Community Manager, has just joined the community party! <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:16.039952996474735% 71.97414806110459%;background-size:5418.75% 5418.75%" data-codepoints="1f389"></span></span>  She will be your "go to" for all things Education helping students and teachers utilize GitHub and push the boundaries of using open source within the classroom. Her career started in software engineering then led onto community management where she was selected as a finalist for Red Hat’s 2020 Women in Open Source Award <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:20.035252643948294% 57.99059929494712%;background-size:5418.75% 5418.75%" data-codepoints="1f3c6"></span></span> . Her ongoing passion is getting open source participation into the school curriculum and is extremely excited to help teachers facilitate that. She loves urban homesteading and is aiming to become more self-sufficient which has her learning new skills like sewing 🪡 , knitting, foraging, growing fruits and vegetables <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:14.042303172737954% 55.99294947121034%;background-size:5418.75% 5418.75%" data-codepoints="1f353"></span></span> <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://gaoflciahikhligngeccdecgfjngejlh/emoji-data/sheet_apple_32.png);background-position:83.96004700352526% 34.01880141010576%;background-size:5418.75% 5418.75%" data-codepoints="1f954"></span></span>  and general DIY.

**Comment by melvincwng** [comment]
Shoutout to the entire GitHub Team over here for doing a wonderful job in moderating the discussions and maintaining this wonderful community here :)

Thank you so much 👍🏻

**Reply by tuves** [reply]
Thanks for being here, @melvincwng! It's awesome seeing folks like you make a difference by helping out other GitHub Community members.

**Reply by melvincwng** [reply]
@tuves , the pleasure is mine! Happy to help out fellow GitHub members 😊

**Reply by Kagbara** [reply]
This will be a life time experience. Thank you to the amazing team for your endless and selfless support. 

We are getting better because of you guys.

**Reply by bethelben** [reply]
hello am new here and I don't even know where to start. some assistance will be nice. am an easy going person and I also a fast learner, looking forward to familiarizing here with you. big thanks.

**Reply by ossanna16** [reply]
Hi there @bethelben and welcome to our GitHub Community! Please check out the [New to GitHub category](https://github.com/orgs/community/discussions/categories/new-to-github) and let me know if it's helpful to you.

**Reply by the-officialjosh** [reply]
Awesome Work guys

**Reply by ossanna16** [reply]
Thank you for the kind words @the-officialjosh 🙂

**Reply by Cayth-X** [reply]
thx

**Reply by Tara8811** [reply]
Hello everyone I've got no clue what this is all about but I'm here wanting to no so be great to find out what this is all about thank u

**Reply by santosh370** [reply]
Hi all, I have also starting to share my knowledge through GIthub. My profile is [Santosh370](https://github.com/santosh370). Please review and share your feedback. I am eager to learn and contribute.

Thanks!

**Reply by ossanna16** [reply]
Hi there @santosh370! Please [introduce yourself](https://github.com/orgs/community/discussions/23204) in our Introduction Discussion. There's a better chance people will see your message there 🙂

**Reply by swayze318** [reply]
I also am brand new to github & have no idea what this platform is used for if its writing code or something used for developing. If the community would help with any info Id greatly appricate it. I have no prior experiace in coding or developing any and all software

**Reply by ossanna16** [reply]
Welcome @swayze318 👋 Check out [GitHub Skills](https://skills.github.com) for some helpful tutorials which will hopefully answer some of your questions 🙂

**Comment by radar-blip** [comment]
Hello Team!! 👍🏼

**Reply by tuves** [reply]
Hello @linuxlawson! I've also seen you around helping out others with their questions. Thanks for being here with us! :)

**Reply by radar-blip** [reply]
You're welcome. I'm not much help but I help if I can. 
Have a great day!

**Reply by libos1** [reply]
can i easily 
learn github when i don't have idea about it

**Reply by Kashumy** [reply]
Does anyone know how to change the text color in readme.md
💀

**Reply by HttpMeech** [reply]
this has always been a dream of mine, coming from ground zero with all this, and im new to this, anyone that gives me any advice or cared to help me navigate around github and learn this amazing craft would be a blessing and well apprieciated!

**Comment by Jimmyzinger** [comment]
Thank you all for being here. Much appreciated, is this a new thing GitHub is sponsoring, or is this Welcome Community with its large talent pool, organization and determination  doing this for a project. Hate to say thank you and be cynical in a small way about the motivation, but that’s just me some days. Glad to be here and plan to spend some hours searching and just traipsing around this gitverse. Peace.

**Reply by ossanna16** [reply]
Welcome @Jimmyzinger!

**Reply by FabianaCampanari** [reply]
Hello everybody, so glad that this community is available!

**Comment by MathiasReker** [comment]
Hello! Welcome to GitHub! 👋🏻

**Reply by Whiskey992** [reply]
Tapwe tawow github miyason noosim lol

**Reply by onyebuchinzube247** [reply]
I have been here for long time ago, but I left because I was ill and can't continue then , so I am happy to here again, please who will take my hands?

**Comment by naji0329** [comment]
Hello everyone. I also hope to help other!

**Reply by ossanna16** [reply]
Welcome @NAJI2329! Do you have a preferred topic you'd like to help people with?

**Comment by PGautam27** [comment]
Starting a community is really good way to find our fellow minded people. And not to forget we get a lot of help. And I wish to of help to the community as well. 😁

**Reply by tuves** [reply]
Glad to hear it, @PGautam27! We're happy to have you here with us.

**Comment by Psianturi** [comment]
Hi Github Community Team.. Thank you and welcome
:)

**Reply by ossanna16** [reply]
Welcome @Psianturi, so happy you found our community 🙂

**Comment by Whiskey992** [comment]
Glad to be back how about you all

**Reply by ossanna16** [reply]
Hi @Whiskey992, glad to have you here 🙂

**Comment by mahamat-abdelkerim-abakar** [comment]
Hi community team

**Reply by tuves** [reply]
Hi @mahamat-abdelkerim-abakar! Welcome to the GitHub community! 👋🏽

**Comment by melissapalhano** [comment]
Hiyaaa Github Comunity Team!!!❤

**Reply by ossanna16** [reply]
Hi there @melpalhano and welcome to our community ✨

**Reply by Jimmyzinger** [reply]
Hello @melpalhano.

Hi to everyone here as well. I don't come here often, but I really should spend more time. I am trying to educate myself and get certified in Network Security. I could use some pointers on ways to achieve the best possible certification that may provide an entrance to the field. Thank you in advance for your time and consideration in giving me some direction, if any.

**Comment by himanshu1221** [comment]
Hellooooo github team 👋😇

**Reply by ossanna16** [reply]
Hi there @himanshu1221, happy you found our community 💯

**Reply by Kashumy** [reply]
Hey @himanshu1221

**Comment by shiddugmail** [comment]
Hello Everyone,

Good evening/Good morning !

Hope you are all doing well. Great to see detailed information about each team member. @ossanna16 

Regards,
Shiddu

**Reply by ossanna16** [reply]
Welcome @shiddugmail 🙂

**Comment by SilentGlasses** [comment]
Hi team, great job keeping the place clean and health for all that come by.

**Reply by ossanna16** [reply]
Thank you for the kind words @SilentGlasses and welcome to our community 🙂

**Comment by nitinraturi** [comment]


**Reply by queenofcorgis** [reply]
Hi @nitinraturi, we’re glad you’re here 😀 welcome! 

Thanks for sharing your projects and socials with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by mrdrdocs** [comment]
We have onboarded dependabot to one of the Repo and ignored couple of packages for not to look for version updates. Still Pull Requests are created for ignored dependencies @dependabot.
Ex: 
updates:
  - package-ecosystem: "nuget"
    directory: "/" #root
    schedule:
      interval: "daily"
    ignore:
      - dependency-name: "Polly"
      - dependency-name: "Polly.*"
      - dependency-name: "Moq"
      - dependency-name: "FluentAssertions"
    
We see PRs are created for Polly, Moq, FluentAssertions and Polly.* [ Polly.Extensions.Http ].
Please provide inputs on this to fix the issue!

**Reply by ossanna16** [reply]
Hi there @mrdrdocs! Welcome to our community and thank you for your question! Please create a separate thread for your question in the right category as you will have a higher chance of receiving a response that way. Cheers!

**Comment by alokVerma749** [comment]


**Reply by ossanna16** [reply]
Hi @alokVerma749, we’re glad you’re here 😀
Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Comment by Kashumy** [comment]
I'm new does anyone know how to change the color text in readme.md

> `??? `

**Reply by ossanna16** [reply]
Hi there @Kashumy! [This article](https://www.w3schools.io/file/markdown-styles/) might be helpful to you. Let me know how it goes!

**Reply by Kashumy** [reply]
Thank  you .  But that didn't help me too much. At least giving pictures works 👍. If you want to help me with this, you can send some repositories on this topic. Thank you very much for everything :).[ @ossanna16 ] 
**Thanks 👍**

**Comment by alokVerma749** [comment]
hi, @ossanna16 the article is great it says to `Write inline HTML code in markdown content`. But the GitHub markdown parser doesn't support inline HTML attributes anymore.

**Reply by ossanna16** [reply]
Thanks for letting me know! Do you know of an alternative solution?

**Reply by alokVerma749** [reply]
No, I am also searching for the same type of issue.
Actually, I am searching for a way to create a link in markdown that opens up in a new tab(no HTML only markdown)

**Reply by Kashumy** [reply]
Thanks, but colored text can be placed using markdown, just upload a photo. 
@alokVerma749! & @ossanna16!
. Maybe it's not what we want, but you can upload a transparent photo and text on it. It doesn't require html in .md
😉

**Reply by Kashumy** [reply]
If any of you don't know how to insert an image in markdown.md, please visit my profile and copy the program. Be sure to specify the path to your md and put it in place of your_path

**Reply by ossanna16** [reply]
Thank you for the great tips @Kashumy!

**Comment by chimcuccu258** [comment]
Have a good day!

**Reply by ossanna16** [reply]
And you @chimcuccu258 🙂

**Reply by ArthurGomesp** [reply]
👍

**Reply by samuelvictor-444** [reply]
THANKS A LOT

**Comment by JudsonOrum** [comment]
Hello

**Reply by ossanna16** [reply]
Hi there @JudsonOrum and welcome to the GitHub Community!

**Comment by opticaldrive** [comment]
Thank you to the GitHub team and community!

**Reply by opticaldrive** [reply]
Have a great day!

**Reply by ossanna16** [reply]
Thank you for being part of our community @V205Arduino! Have a great day as well 🙂

**Comment by AnuragNishad12** [comment]
hii everyone ,
I'm getting an error all over my github repository 
this one 
<img width="943" alt="image" src="https://github.com/community/community/assets/124987038/1a2e7ee6-344d-4c94-ab12-17ad3b5a0924">
Can anyone knows how to solve it ?
Please help me

**Reply by ossanna16** [reply]
Hi there @AnuragNishad12! Please post your question in the [Repositories category](https://github.com/orgs/community/discussions/categories/repositories) as it's more likely for you to receive help there. Thank you 🙂

**Comment by amitvtcbb** [comment]
Hello GitHub Team, 

First of all Thank you all team member around helping out others with their questions.

Actually using from last 1 year but not so active. But after using it there are so much confusion and query.
Actually I don't even know where to start. if some assistance will be nice, if I will get on some topic. 

I teach programming as well as theoretical subject of computer science. There are few question

1  My question is how to use one repo for weekly assignment. can you please tell me step by step process?

2  Also once I create repo and generate assignment, if any changes made inside the repo, can it be reflected on student assignment? 

3. I want student comment how can I download it?

**Reply by ossanna16** [reply]
Hi there @amitvtcbb and welcome to our community. Please post your question in the [repositories category](https://github.com/orgs/community/discussions/categories/repositories) as it is more likely you will receive help there.

Thank you!

**Comment by sheri-na** [comment]
Hi! It is nice to meet you guys!  Thank you for all the hard work you guys do! I loved the video you recommended, @ghostinhershell, I added it to my playlist. It literally couldn't have come at the right time, I really needed this. @samus-aran,  I love homesteading too , if I had the opportunity I would definitely bite the bullet! I think it is a great opportunity to become more self-sufficient and learn from the ground up.

**Reply by ossanna16** [reply]
Thank you for the kind words @minamin33, our team is full of lovely people ❤️

**Comment by martcpp** [comment]
Shout our to all the great team

**Reply by ossanna16** [reply]
Thanks so much @martcpp 🙂

**Comment by LaoSun2023** [comment]
hey there,I am a beginner and I need to develop a mobile app that is compatible with both iOS and Android operating systems. The app should have a front-end UI development based on lock screen and screensaver (widget), control over the device's microphone and speaker, and integration with an external voice recognition engine with performance optimization. Can you recommend a development platform and programming language that is easy to learn and offers good runtime efficiency?

**Reply by ossanna16** [reply]
Hi there @LaoSun2023 and welcome to our community. Please post your question in the [Programming Help category](https://github.com/orgs/community/discussions/categories/programming-help) as it is more likely you will receive help there.

Thank you!

**Reply by LaoSun2023** [reply]
Thanks so much @ossanna16

**Reply by ossanna16** [reply]
Happy to help 🙂

**Comment by BDawg3327** [comment]
Who are you

**Comment by tricsi88363** [comment]
thanks for being here, although one more two help would be helpful;

**Comment by lotirium** [comment]
Hello team!!!!

**Reply by ossanna16** [reply]
Welcome to the GitHub Community @lotirium 👋

**Comment by brunoblaise** [comment]
Hi @everyone Happy Holidays, nice to meet GitHub community Team and everyone else.

Thanks😊

**Reply by ossanna16** [reply]
Happy new year @brunoblaise! Nice to meet you too 🙂

**Reply by ozonejnr** [reply]
Kudos to @everyone making this community a success, I am new around here and would love to learn faster so as to also help this great community in my little ways.

**Reply by ossanna16** [reply]
Welcome @ozonejnr, glad you found the GitHub Community 👋
